modules/Core/queries/debug_link.php

```diff
@@ -179,10 +179,36 @@
     $logs[$type] = file_exists($file_path) ? Util::readFileEnd($file_path, $max_bytes = 10_000) : '';
 }
 
+$user_data = [];
+if ($user->isLoggedIn()) {
+    $user_groups = [];
+    foreach ($user->getAllGroupIds() as $group_id) {
+        $user_groups[] = $group_id;
+    }
+
+    $user_integrations = [];
+    foreach ($user->getIntegrations() as $integrationUser) {
+        $user_integrations[$integrationUser->getIntegration()->getName()] = [
+            'username' => $integrationUser->data()->username,
+            'identifier' => $integrationUser->data()->identifier,
+            'verified' => $integrationUser->data()->verified ? true : false
```

Comment:
We probably don't need this ternary if "verified" returns a bool already

Reply:
Solved